OpenRA.Mods.Common/Traits/Capturable.cs

```diff
@@ -50,11 +50,11 @@ public bool CanBeTargetedBy(Actor captor, Player owner)
 		}
 	}
 
-	public class Capturable : INotifyCapture
+	public class Capturable : ConditionalTrait<CapturableInfo>, INotifyCapture
 	{
-		public readonly CapturableInfo Info;
+		public readonly new CapturableInfo Info;
```

Comment:
Remove this - the `ConditionalTrait` already provides an `Info` field and there is no benefit to overriding it.